helm/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
 name: zaakafhandelcomponent
 description: A Helm chart for installing Zaakafhandelcomponent
-version: 0.1.0
+version: 0.2.0
 appVersion: "latest"
-dependencies:
-  - name: solr
-    version: ~6.2.3
-    repository: https://charts.bitnami.com/bitnami
+#dependencies:
```

Comment:
Better to just remove this altogether? If we can avoid it we do not want uncommented code in our repos since that only leads to confusion. ;-)